exhibitor/models.py

```diff
@@ -19,10 +19,36 @@ def __str__(self):
         return self.username
 
 class Moc(models.Model):
+    CASTLE = 'Castle/Starożytność'
+    ARCHITEKTURA = 'Architektura'
+    KOMIKS = 'Komiks/Anime/Manga'
+    MILITARIA = 'Militaria'
+    POJAZDY_MINIFIG = 'Pojazdy Minifig'
+    POJAZDY_TECHNIC = 'Pojazdy technic'
+    SPACE = 'Space/Sci-fi'
+    KOLEJNICTWO = 'Kolejnictwo'
+    CATEGORY = [
+        ('Castle/Starożytność', 'Castle/Starożytność'),
+        ('Architektura', 'Architektura'),
+        ('Komiks/Anime/Manga', 'Komiks/Anime/Manga'),
+        ('Militaria', 'Militaria'),
+        ('Pojazdy Minifig', 'Pojazdy Minifig'),
+        ('Pojazdy technic', 'Pojazdy technic'),
+        ('Space/Sci-fi', 'Space/Sci-fi'),
+        ('Kolejnictwo', 'Kolejnictwo'),
+    ]
+    FIRST_SIZE = '0,5 stołu'
+    SECOND_SIZE = '1 stół'
+    THIRD_SIZE = '2 stoły'
+    SIZE = [
+        ('0,5 stołu', '0,5 stołu'),
+        ('1 stół', '1 stół'),
+        ('2 stoły', '2 stoły'),
+    ]
     author = models.ForeignKey(User, on_delete=models.CASCADE)
     title = models.CharField(max_length=20)
-    category = models.CharField(max_length=30, choices=['kat1''kat2''kat3''kat4''kat5']),
-    size = models.CharField(max_length=20, choices=['1','2','3']),
+    category = models.CharField(max_length=20, choices=CATEGORY, default=CASTLE)
+    size = models.CharField(max_length=20, choices=SIZE, default=FIRST_SIZE)
```

Comment:
possibly better to change to float (you could use math later on to simply add  tables)